lib/commonBindings.js

```diff
@@ -156,20 +156,22 @@ function processMeasureGroup(device, apikey, previous, current, index) {
  * @param {Object} device           Device object containing all the information about a device.
  * @param {String} message          UL payload.
  */
-function multipleMeasures(apiKey, device, message) {
+function multipleMeasures(apiKey, device, messageStr, message) {
     let updates = [];
-    const messageStr = message.toString();
     let parsedMessage;
 
     config.getLogger().debug('Processing multiple measures for device [%s] with apiKey [%s]', device.id, apiKey);
 
+    // Parse Message
     try {
         parsedMessage = ulParser.parse(messageStr);
     } catch (e) {
-        config.getLogger().error(context, 'MEASURES-003: Parse error parsing incoming message [%]', messageStr);
```

Comment:
Given that now the parser error is not possible, MEASURE-003 is no longer used as error. It should be removed from documentation at docs/operations.md

Reply:
done in https://github.com/telefonicaid/iotagent-ul/pull/532/commits/b77ad41a6c5427919b3afbfb666a70841d33bca2